packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -554,6 +544,44 @@ export default class SalesforceCRMService implements CRM {
     }
   }
 
+  private async getContactOrLeadFromEmail({
```

Comment:
In both places this is being used, we prioritize returning the contact over the lead.